HHS/Head-Start-TTADP
# [TTAHUB-1680] Single or multiple recipients filter

## Description of change

Add filter to allow viewing reports with single or multiple recipients.

**NOTE:** Patrice says we don't need to worry about OE reports for this filter. She also agreed to a name change for the filter of 'Single or multiple recipients'.

## How to test

**Test on:** AR Landing, Regional Dashboard, Resource Dashboard and Goals (not RTR).
- Create filter to show single reports and verify. 
- Create filter to show multiple and verify.

We should count recipients with the same UEI number but different names as the same recipient (single).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1680


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
